### PR TITLE
Added nysomh_mentalhealth pipeline function

### PR DESCRIPTION
**Pull request by rubenvecg**
Part of #359

### CODE REVIEW

File: facdb/pipelines.py [hunk 1]
```diff
@@ -514,8 +514,18 @@ def nysoasas_programs(df: pd.DataFrame = None):
 
 
 @Export
+@Function1B(
+    street_name_field="parsed_sname",
+    house_number_field="parsed_hnum",
+    borough_field="program_borough",
```
Comment (mgraber) [inline facdb/pipelines.py:520]: Is this a field in the input data? I don't see it. You can pass county in as borough, though.

Comment (rubenvecg) [reply facdb/pipelines.py]: I thought it had a borough and then forgot to remove it. I fixed it now

Comment (rubenvecg) [reply facdb/pipelines.py]: Is county ok for a borough field or maybe we could make a simple function that returns the corresponding borough? Like 
```
if county == 'Richmond'
return 'Staten Island'
```

Comment (mgraber) [reply facdb/pipelines.py]: The geosupport python bindings do that already: https://github.com/ishiland/python-geosupport/blob/master/geosupport/config.py#L12-L19

**Review by mgraber — APPROVED** [review]

**Pull request merged**